core/startos/src/auth.rs

```diff
@@ -91,29 +91,41 @@ pub fn auth<C: Context>() -> ParentHandler<C> {
                 .with_metadata("login", Value::Bool(true))
                 .no_cli(),
         )
-        .subcommand("login", from_fn_async(cli_login).no_display())
+        .subcommand(
+            "login",
+            from_fn_async(cli_login)
+                .no_display()
+                .with_about("Log in to StartOS server allowing use of tools such as side-loading services via CLI"),
```

Comment:
This is maybe a bit too verbose - sideloading is just one of dozens of authenticated endpoints, and calling it out specifically here assumes too much about the intent of the user